packages/tree-widget/src/TreeWidget.ts

```diff
@@ -13,12 +13,17 @@ import { UiError, getClassName } from "@bentley/ui-abstract";
  */
 export class TreeWidget {
   private static _i18n?: I18N;
+  private static _initialized: boolean;
 
   /**
    * Called by IModelApp to initialize the Tree Widget
    * @param i18n - The internationalization service created by the IModelApp.
    */
   public static async initialize(i18n: I18N): Promise<void> {
```

Comment:
could you make this optional(the arg), and default to IModelApp.i18n if i18n is undefined?